css/_sidenavs.scss

```diff
@@ -110,3 +110,14 @@
     margin-right: 10px!important;
     margin-left: 5px!important;
 }
+
+md-sidenav {
```

Comment:
Yes. This commit tries to override those built-in rules within the styleguide. And it seems those rules are added dynamically when `ngMaterial` is injected. However, new classes will suffice for now, so I'll commit another version.